contracts/StablecoinConverter.sol

```diff
@@ -97,71 +135,32 @@ contract StablecoinConverter is EpochTokenLocker {
         emit OrderCancelation(msg.sender, id);
     }
 
-    function freeStorageOfOrder(
-        uint[] memory id
-    ) public {
-        for (uint i = 0; i < id.length; i++) {
-            require(orders[msg.sender][id[i]].validUntil + 1 < getCurrentBatchId(), "Order is still valid");
-            delete orders[msg.sender][id[i]];
+    /** @dev a user facing function used to delete expired orders
+      * @param ids referencing the indices of user's orders to be deleted
+      */
+    function freeStorageOfOrder(uint[] memory ids) public {
+        for (uint i = 0; i < ids.length; i++) {
+            require(orders[msg.sender][ids[i]].validUntil + 1 < getCurrentBatchId(), "Order is still valid");
+            delete orders[msg.sender][ids[i]];
         }
     }
 
-    function tokenAddressToIdMap(address addr) public view returns (uint16) {
-        return IdToAddressBiMap.getId(registeredTokens, addr);
-    }
-
-    function tokenIdToAddressMap(uint16 id) public view returns (address) {
-        return IdToAddressBiMap.getAddressAt(registeredTokens, id);
-    }
-
-    function getEncodedAuctionElements() public view returns (bytes memory elements) {
-        if (allUsers.size() > 0) {
-            address user = allUsers.first();
-            bool stop = false;
-            while (!stop) {
-                for (uint i = 0; i < orders[user].length; i++) {
-                    Order memory order = orders[user][i];
-                    elements = elements.concat(encodeAuctionElement(
-                        user,
-                        getBalance(user, tokenIdToAddressMap(order.sellToken)),
-                        order
-                    ));
-                }
-                if (user == allUsers.last) {
-                    stop = true;
-                } else {
-                    user = allUsers.next(user);
-                }
-            }
-        }
-        return elements;
-    }
-
-    mapping (uint16 => uint128) public currentPrices;
-    PreviousSolutionData public previousSolution;
-
-    struct PreviousSolutionData {
-        uint32 batchId;
-        TradeData[] trades;
-        uint16[] tokenIdsForPrice;
-        address solutionSubmitter;
-        uint256 feeReward;
-        uint256 objectiveValue;
-    }
-
-    struct TradeData {
-        address owner;
-        uint128 volume;
-        uint16 orderId;
-    }
-
+    /** @dev a solver facing function called for auction settlement
+      * @param batchIndex index of auction solution is referring to
+      * @param owners array of addresses corresponding to touched orders
+      * @param orderIds array of order ids used in parallel with owners to identify touched order
+      * @param volumes executed buy amounts for each order identified by index of owner-orderId arrays
+      * @param prices prices for touched tokens indexed by next parameter
+      * @param tokenIdsForPrice ordered list of touched token ids
```

Comment:
If we have the documentation here, maybe we can replace the comments in code.

btw, 
```
// fee token id not required since always 0
```
is no longer valid. Would you mind cleaning it in this PR?